packages/graphql/tests/tck/connections/connect-or-create/connect-or-create.test.ts

```diff
@@ -463,8 +463,8 @@ describe("Create or Connect", () => {
 
                 interface ActedIn @relationshipProperties {
                     id: ID! @id
-                    createdAt: DateTime! @timestamp(operations: [CREATE])
-                    updatedAt: DateTime! @timestamp(operations: [UPDATE])
+                    createdAt: DateTime @timestamp(operations: [CREATE])
```

Comment:
You should be able to leave these as non-nullable!

```suggestion
                    createdAt: DateTime! @timestamp(operations: [CREATE])
```